cluster/manifests/logging-agent/daemonset.yaml

```diff
@@ -112,11 +112,11 @@ spec:
 
         resources:
           limits:
-            cpu: 200m
-            memory: 200Mi
+            cpu: 100m
+            memory: 150Mi
```

Comment:
This definitely won't work, I've seen in crashing in a couple of clusters with the existing `200Mi` limit. There's another PR (https://github.com/zalando-incubator/kubernetes-on-aws/pull/1486) to increase it instead.

Reply:
I adjusted the resource to the ones from the other PR and closed the other PR